tableau_utilities/scripts/cli.py

```diff
@@ -162,6 +162,7 @@
 parser_datasource.add_argument('-fe', '--filter_extract',
                                help='Deletes data from the extract based on the condition string provided. '
                                     """E.g. "CREATED_AT" < '1/1/2024'""")
+parser_datasource.add_argument('-ci', '--column_init', action='store_true',  help='Alters file to add all columns seen in Metadata records to column objects.')
```

Comment:
```suggestion
parser_datasource.add_argument('-ci', '--column_init', action='store_true',  help="Adds Columns from all Metadata Records, if they don't already exist.")
```
Think the wording here could be updated a bit.